scripts/react-native-xcode.sh

```diff
@@ -194,14 +194,12 @@ else
   if [[ $EMIT_SOURCEMAP == true ]]; then
     EXTRA_COMPILER_ARGS="$EXTRA_COMPILER_ARGS -output-source-map"
   fi
-  HBC_FILE="$CONFIGURATION_BUILD_DIR/$(basename $BUNDLE_FILE)"
```

Comment:
@alloy, decided to remove this variable as it's essentially `BUNDLE_FILE` at this point. 

For reference, `BUNDLE_FILE` is `CONFIGURATION_BUILD_DIR/main.jsbundle`.